savant/deepstream/buffer_processor.py

```diff
@@ -458,16 +455,24 @@ def prepare_element_input(self, element: PipelineElement, buffer: Gst.Buffer):
                             ),
                         )
 
+                    else:
+                        self.logger.warning(
+                            'The object (%s.%s, bbox %s) is an orphan (no parent object is assigned). It is a non-typical case: the object should either have the frame or an ROI object as a parent.',
```

Comment:
The line is too long. Split it to multiple lines.

Also I suggest to add source ID, frame PTS and IDX to the log.

Reply:
@tomskikh add proposed info

Reply:
Added.

`The object (VehicleDetector.vehicle, bbox (640, 496, 790, 609)) of a frame source-2/766666666 with IDX 28 is an orphan (no parent object is assigned). It is a non-typical case: the object should either have the frame or an ROI object as a parent.`